Dashboard/ViewModels/MainWindowViewModel.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Avalonia.Controls;
+using CommunityToolkit.Mvvm.ComponentModel;
+using Dashboard.Models;
+using Dashboard.Utils;
+using Dashboard.Views;
+using Microsoft.Extensions.DependencyInjection;
+
+
+
+namespace Dashboard.ViewModels
+{
+    public partial class MainWindowViewModel : ViewModelBase, IDisposable
+    {
+        /// <summary>
+        /// Observable variables that utilize ObservableObject for reactive commands.
+        /// </summary>
+        [ObservableProperty]
+        private object? _currentPage;
```

Comment:
nit: spacing under this line.

does this pass formatting? maybe we need to add a rule to ensure there are spaces

Reply:
answer: yes it does pass formatting.

is anyone against me adding a rule to make sure we have spacing between our member vars?

Reply:
I'm not opposed to any styling changes as long as we're consistent.